README.md

```diff
@@ -1 +1,18 @@
-# Cere POS Network Node
+#Cere Node with PoS consensus
+
+## Try it out
+
+Follow the [instructions](https://github.com/Cerebellum-Network/validator-instructions#how-to-become-a-validator) to get the Node up and running.
```

Comment:
I think we need to highlight that this is the instructions about how to become a Validator.
We should create another section - how to up and run the Network from scratch and add this reference here.

Reply:
Let's rename the section `How to become validator`